src/components/DropdownNavItem.vue

```diff
@@ -1,12 +1,12 @@
 <template>
   <span>
-    <b-dd-item v-if="isLocal" :to="target" exact>
+    <b-dd-item v-if="isLocal" :to="target" exact v-b-tooltip.bottom :title="(!labelIfMobile && !$isMobile()) ? label : ''">
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Replace `·v-if="isLocal"·:to="target"·exact·v-b-tooltip.bottom·:title="(!labelIfMobile·&&·!$isMobile())·?·label·:·''"` with `⏎······v-if="isLocal"⏎······:to="target"⏎······exact⏎······v-b-tooltip.bottom⏎······:title="!labelIfMobile·&&·!$isMobile()·?·label·:·''"⏎····`](https://app.codacy.com/manual/WFCD/warframe-hub/pullRequest?prid=5222127)